### PR TITLE
DOM-35895 DCO Deployment

### PR DESCRIPTION
**Pull request by adp312**

### CODE REVIEW

File: deploy/helm/distributed-compute-operator/values.yaml [hunk 1]
```diff
@@ -111,5 +111,5 @@ nameOverride: ""
 fullnameOverride: ""
 
 mpi:
-  initImage: "quay.io/domino/distributed-compute-operator-mpi-init:0.0.0"
-  syncImage: "quay.io/domino/distributed-compute-operator-mpi-sync:0.0.0"
+  initImage: quay.io/domino/distributed-compute-operator-mpi-init:main
+  syncImage: quay.io/domino/distributed-compute-operator-mpi-sync:main
```
Comment (JoshBroomberg) [inline deploy/helm/distributed-compute-operator/values.yaml:115]: WHy don't these use the same split-up format as the one above?

**Review by JoshBroomberg — APPROVED** [review]
One nit, but looks good.

File: deploy/helm/distributed-compute-operator/templates/deployment.yaml [hunk 2]
```diff
@@ -69,10 +69,10 @@ spec:
             - --istio-enabled
             {{- end }}
             {{- with .Values.mpi.initImage }}
-            - --mpi-init-image={{ . }}
+            - --mpi-init-image="{{ .registry }}/{{ .repository }}:{{ .tag }}"
```
Comment (JoshBroomberg) [inline deploy/helm/distributed-compute-operator/templates/deployment.yaml:72]: I believe you can do `{{ include "common.imagename" .Values.mpi.initImage }}`

Comment (JoshBroomberg) [reply deploy/helm/distributed-compute-operator/templates/deployment.yaml:72]: This is the point of using the split up OCI format

File: deploy/helm/distributed-compute-operator/values.yaml [hunk 3]
```diff
@@ -57,11 +57,11 @@ prometheus:
   namespaceLabels: {}
 
 image:
-  registry: ghcr.io
+  registry: quay.io
   repository: dominodatalab/distributed-compute-operator
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: ""
+  tag: main
```
[the remark this answers is not in the document]
Comment (JoshBroomberg) [reply deploy/helm/distributed-compute-operator/values.yaml:64]: Main. No tag will use latest - which is not what we want

**Review by JoshBroomberg — COMMENTED** [review]
LIttle nit. Otherwise I think we should merge this and then work out a solution for the image sync problem (it is solved in an ok way for now)

File: deploy/helm/distributed-compute-operator/templates/deployment.yaml (hunk 2 above)
Comment (adp312) [reply deploy/helm/distributed-compute-operator/templates/deployment.yaml:72]: should it be "common.images.image"? https://artifacthub.io/packages/helm/bitnami/common#images

**Comment by adp312** [comment]
Tested everything. Will commit next Monday.

**Pull request merged**